src/hotspot/cpu/x86/stubGenerator_x86_64_arraycopy.cpp

```diff
@@ -215,7 +215,7 @@ void StubGenerator::array_overlap_test(address no_overlap_target, Label* NOLp, A
   __ cmpptr(to, from);
   __ lea(end_from, Address(from, count, sf, 0));
   if (NOLp == nullptr) {
-    ExternalAddress no_overlap(no_overlap_target);
+    RuntimeAddress no_overlap(no_overlap_target);
```

Comment:
I think this is actually a jump within the same buffer. Can we use an AddressLiteral here and rely on this being encoded with a PC-relative jump?

Reply:
`relic_none` is treated as unreachable and will force using lea():
https://github.com/openjdk/jdk/blob/master/src/hotspot/cpu/x86/assembler_x86.cpp#L12915

Reply:
And `runtime_call_type' will be reachable if inside CodeCache:
https://github.com/openjdk/jdk/blob/master/src/hotspot/cpu/x86/assembler_x86.cpp#L12897

Reply:
Ok, so the PR looks good then.